src/client/blocks-ext.js

```diff
@@ -256,9 +256,20 @@ RPCInputSlotMorph.prototype.methodSignature = function () {
     if (rpc) {
         this.fieldsFor = JSON.parse(this.getURL('/rpc/' + rpc));
 
+        var privateFields = {};
```

Comment:
I am not sure if I like this approach for serving the service info. Rather than adding fields prefixed with a `_` to encode additional information for the entire service, it might be better to change the overall signature in a more easily parsed and intuitive way such as returning the following for the `'/rpc/' + rpc` endpoint:

```
[
  {
    "name": "getTerrainMap",
    "arguments": ["latitude", "longitude", "zoom", "width", "height"],
    "optional": [],
    "deprecated": false
  }
]
```

Reply:
should we have that in another PR?

Reply:
Yeah, that is probably a good idea.

Reply:
I created an issue for this https://github.com/NetsBlox/NetsBlox/issues/1620